feedbag/template/template_test.go

```diff
@@ -1,9 +1,7 @@
-package feedbag_test
+package template
```

Comment:
lets keep this same `*_test` pattern and make this `template_test` and actually import the `template` package to test against